src/views/Signup.js

```diff
@@ -236,6 +283,15 @@ export default class Signup extends Component {
                   {this.state.passwordConfirmation.message}
                 </Form.Control.Feedback>
               </Form.Group>
+
+              <Form.Group controlId="signupCollabRolese">
+                <Form.Control as="select">
+                  {roleOptions.map((option) => (
```

Comment:
Add a blank option and a placeholder in the input of 'Role' rather than defaulting to the first option:

![image](https://user-images.githubusercontent.com/13375042/140167244-03062537-5bcb-4153-9b95-560412cc265c.png)